support-lambdas/acquisitions-firehose-transformer/src/main/scala/com/gu/acquisitionFirehoseTransformer/Lambda.scala

```diff
@@ -32,7 +32,11 @@ case class AcquisitionWithRecordId(val acquisition: AcquisitionDataRow, recordId
 
 object Lambda extends LazyLogging {
 
-  def handler(event: KinesisFirehoseEvent, context: Context): KinesisAnalyticsInputPreprocessingResponse = {
+  def handler(event: KinesisFirehoseEvent): KinesisAnalyticsInputPreprocessingResponse = {
+    processEvent(event, new AnnualisedValueServiceImpl())
+  }
+
+  def processEvent(event: KinesisFirehoseEvent, avService: AnnualisedValueServiceWrapper) = {
```

Comment:
We've factored out a `processEvent` method so we can make use of dependency injection in our tests to provide a simple mock av service